### DISCUSSION TITLE
Error while calling api provisionSmartStartNode: The  Error while calling api provisionSmartStartNode: The controller does not support the SmartStart feature (ZW0207)  does not support the SmartStart feature (ZW0207)

### DISCUSSION THREAD

**Discussion by adrianocr** [zwave-js/zwave-js#5107] [Q&A] [answered]
I'm using a Nortek HUSBZB-1 zwave/zigbee stick in home assistant and also connected to ZWave JS. 

I got a new outdoor lights controller (Enbrigthen zwave outdoor smart switch) and when trying to add the device to zwave js I get the error "Error while calling api provisionSmartStartNode: The  Error while calling api provisionSmartStartNode: The controller does not support the SmartStart feature (ZW0207)  does not support the SmartStart feature (ZW0207)"

Does that mean that my zwave stick isn't compatible with the zwave version of the device or is something else going on? I have several other zwave devices connected and none of them seem to be a problem except this one.

**Comment by AlCalzone** [comment, marked as answer]
No this means you cannot use SmartStart (hands-free inclusion via QR code) with that controller and need to use the classic inclusion method of pushing buttons (and entering device PIN).

I guess HA should catch that error and show a better error message.
@raman325

**Reply by raman325** [reply]
I'm unclear why the repeating happens so we can look into that, but if we want to improve the error message IMO it should be in the driver. Right now we don't have mappings to each ZwaveError so it would be a bit weird to have different error handling for this particular error type. If we want to do that on the library/HA side, I think it would make sense to map all the errors at that point, but I'd prefer not to do that if I can avoid it 🙂

**Reply by AlCalzone** [reply]
I can definitely look into fixing the repeating part, if it happens in the driver. But this error happens in a very specific situation: HA tries to provision SmartStart without checking if it is supported, and the driver rejects this action.

So either HA should not offer SmartStart on unsupporting controllers or show a better error message than `"error while calling provisionSmartStart: ..."`
This part does not come from the driver AFAIK.

**Reply by raman325** [reply]
Hmm ok I think I'm getting this confused with the firmware update feature because we check capabilities there. Will take a look
